addons/website/controllers/main.py

```diff
@@ -660,6 +660,20 @@ def get_new_page_templates(self, **kw):
                 'title': group_el.text,
```

Comment:
I found this traceback: 
  - Activate the developer mode
  - Go to Website and go to create a new page
  - Go to the "Custom" tab
  => traceback
![image](https://github.com/odoo/odoo/assets/102796037/4a8af5ec-7642-4148-adc2-04885c8ee4c0)

I had this error on a ticket once, this means that you did not set a props correctly. I think you should add a default value for `isCustom` but I am not sure. In debug mode, there is a validation of the props (`validateProps`), which is why it only fails in debug mode (see the doc [here](https://github.com/odoo/owl/blob/master/doc/reference/props.md#props-validation)) :slightly_smiling_face: